calibrate.py

```diff
@@ -472,43 +495,44 @@ def create_pipeline(self):
+                    cam_node = pipeline.createMonoCamera()
+                    xout = pipeline.createXLinkOut()
 
-                cam_node.setBoardSocket(stringToCam[cam_id])
-                cam_node.setResolution(camToMonoRes[cam_info['sensorName']])
-                cam_node.setFps(fps)
+                    cam_node.setBoardSocket(stringToCam[cam_id])
+                    cam_node.setResolution(camToMonoRes[cam_info['sensorName']])
+                    cam_node.setFps(fps)
 
-                xout.setStreamName(cam_info['name'])
-                cam_node.out.link(xout.input)
-            else:
-                cam_node = pipeline.createColorCamera()
-                xout = pipeline.createXLinkOut()
-                
-                cam_node.setBoardSocket(stringToCam[cam_id])
-                sensorName = cam_info['sensorName']
-                print(f'Sensor name is {sensorName}')
-                cam_node.setResolution(camToRgbRes[cam_info['sensorName'].upper()])
-                cam_node.setFps(fps)
-
-                xout.setStreamName(cam_info['name'])
-                cam_node.isp.link(xout.input)
-                if cam_info['sensorName'] == "OV9*82":
-                    cam_node.initialControl.setSharpness(0)
-                    cam_node.initialControl.setLumaDenoise(0)
-                    cam_node.initialControl.setChromaDenoise(4)
-
-                if cam_info['hasAutofocus']:
-                    cam_node.initialControl.setManualFocus(self.focus_value)
-
-                    controlIn = pipeline.createXLinkIn()
-                    controlIn.setStreamName(cam_info['name'] + '-control')
-                    controlIn.out.link(cam_node.inputControl)
-
-            # cam_node.initialControl.setAntiBandingMode(antibandingOpts[self.args.antibanding])
-            xout.input.setBlocking(False)
-            xout.input.setQueueSize(1)
+                    xout.setStreamName(cam_info['name'])
+                    cam_node.out.link(xout.input)
+                else:
+                    cam_node = pipeline.createColorCamera()
+                    xout = pipeline.createXLinkOut()
+
+                    cam_node.setBoardSocket(stringToCam[cam_id])
+                    sensorName = cam_info['sensorName']
+                    print(f'Sensor name is {sensorName}')
+                    cam_node.setResolution(camToRgbRes[cam_info['sensorName'].upper()])
+                    cam_node.setFps(fps)
+
+                    xout.setStreamName(cam_info['name'])
+                    cam_node.isp.link(xout.input)
+                    if cam_info['sensorName'] == "OV9*82":
+                        cam_node.initialControl.setSharpness(0)
+                        cam_node.initialControl.setLumaDenoise(0)
+                        cam_node.initialControl.setChromaDenoise(4)
+
+                    if cam_info['hasAutofocus']:
+                        cam_node.initialControl.setManualFocus(int(self.args.rgbLensPosition[stringToCam[cam_id].name.lower()]))
```

Comment:
This doesn't seem well defined.

Lets first define how the mapping is done and then do remapping once and use that throughout the codebase